change-validation/src/main/scala/com/normation/plugins/changevalidation/snippet/ChangeRequestChangesForm.scala

```diff
@@ -705,7 +706,8 @@ class ChangeRequestChangesForm(
     "#description" #> displaySimpleDiff(diff.modDescription, "description", Text(param.description)))(globalParameterXML)
   }
 
-  private[this] def displayFormDiff[T](diff: SimpleDiff[T], name: String)(implicit fun: T => String = (t: T) => t.toString) = {
+  private[this] def displayFormDiff[T](diff: SimpleDiff[T], rawName: String)(implicit fun: T => String = (t: T) => t.toString) = {
+    val name = StringEscapeUtils.escapeEcmaScript(rawName)
```

Comment:
`name` is supposed to be const everywhere the method is called (e.g. with Scala 3 union types and literal types it would have been obvious from the type that we would'nt need to escape...). I escaped it just in case here